service/middleware.go

```diff
@@ -89,6 +90,15 @@ func RedirectSpruceUsers(w http.ResponseWriter, r *http.Request, redirect string
 	return true
 }
 
+// RedirectIfSpruceSet redirects the user to spruce only if they aren't visiting this page from spruce already and have spruce enabled
```

Comment:
nit: do they have to be on spruce already? Based on your test it looks like this works from evergreen too.

Reply:
I didn't write this comment but it's saying that the function does not do anything if you're already clicking into a spruce page ie clicking https://spruce-staging.corp.mongodb.com/host/i-00ab599007f7dd911 will do nothing

Reply:
oh haha I read "aren't" as "are"
thanks!